src/Search/SearchOptions.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace App\Search;
+
+/**
+ * Search Options Class.
+ */
+class SearchOptions
+{
+    /**
+     * The query string to be searched.
+     *
+     * @var string $queryString
+     */
+    private $queryString = '*';
+
+    /**
+     * Only search for published Information Products.
+     *
+     * @var boolean
+     */
+    private $onlyPublishedInformationProducts = true;
+
+    /**
+     * The current page of results.
+     *
+     * @var integer $currentPage
+     */
+    private $currentPage = 1;
+
+    /**
+     * The maximum results per page.
+     *
+     * @var integer $maxPerPage
+     */
+    private $maxPerPage = 1000;
```

Comment:
Maybe this number could be a 100?

Reply:
As soon as the paginator is added, this could be set back to default 10.